components/Layout/AudioControls.vue

```diff
@@ -1,135 +1,315 @@
+                  </v-icon>
+                </v-btn>
+                <v-btn
+                  icon
+                  :disabled="!getNextItem"
+                  class="mx-1"
+                  @click="setNextTrack"
+                >
+                  <v-icon>mdi-skip-next</v-icon>
+                </v-btn>
+                <v-btn
+                  icon
+                  fab
+                  small
+                  :elevation="isRepeating ? '3' : '0'"
+                  class="mx-1"
+                  :class="isRepeating ? '' : 'text--disabled'"
+                  :color="isRepeating ? 'primary' : undefined"
+                  @click="toggleRepeatMode"
+                >
+                  <v-icon>{{ repeatIcon }}</v-icon>
+                </v-btn>
+              </div>
+              <time-slider v-if="!isFullScreenPlayer" />
+            </div>
+          </v-col>
+          <v-col cols="3" class="d-none d-md-flex align-center justify-end">
             <v-btn class="d-none d-md-inline-flex" icon disabled>
               <v-icon size="18">{{
                 getCurrentItem.UserData.IsFavorite
                   ? 'mdi-heart'
                   : 'mdi-heart-outline'
               }}</v-icon>
             </v-btn>
-          </span>
-          <nuxt-link
-            v-if="getCurrentItem.AlbumArtists[0].Id"
-            tag="span"
-            class="text--secondary text-caption text-truncate mt-md-n2 link"
-            :to="`/artist/${getCurrentItem.AlbumArtists[0].Id}`"
+            <v-tooltip top>
+              <template #activator="{ on, attrs }">
+                <v-btn disabled icon class="mr-2" v-bind="attrs" v-on="on">
+                  <v-icon>mdi-playlist-play</v-icon>
+                </v-btn>
+              </template>
+              <span>{{ $t('queue') }}</span>
+            </v-tooltip>
+            <volume-slider />
+            <transition name="fade-fast" mode="in-out">
+              <v-tooltip v-if="!isFullScreenPlayer" top>
+                <template #activator="{ on, attrs }">
+                  <nuxt-link tag="span" :to="'/playback'">
+                    <v-btn icon class="ml-2" v-bind="attrs" v-on="on">
+                      <v-icon>mdi-fullscreen</v-icon>
+                    </v-btn>
+                  </nuxt-link>
+                </template>
+                <span>{{ $t('fullScreen') }}</span>
+              </v-tooltip>
+            </transition>
+            <v-tooltip top>
+              <template #activator="{ on, attrs }">
+                <v-btn
+                  icon
+                  class="ml-2"
+                  v-bind="attrs"
+                  v-on="on"
+                  @click="stopPlayback"
+                >
+                  <v-icon>mdi-close</v-icon>
+                </v-btn>
+              </template>
+              <span>{{ $t('stopPlayback') }}</span>
+            </v-tooltip>
+            <item-menu :item="getCurrentItem" :absolute="false" :dark="false" />
```

Comment:
This is up for discussion, but I think currently, having this here makes the menu feel a bit too cluttered, how about moving it to the main playback page

Reply:
@Camc314 The idea is to have it as we do in current web at some point, with all the actions quickly accesible. I don't mind really, for mobile design though I would leave it.